recipes/mlflow/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "mlflow" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c2c76d072f4735a229333095506acb3d6ec0305656ee7fc468e9b60a0e8d7b8a
+
+build:
+  number: 0
+  entry_points:
+    - mlflow=mlflow.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - click >=7.0
+    - cloudpickle ==0.6.1
+    - databricks-cli >=0.8.0
+    - docker >=3.6.0
```

Already exists as `docker-py` 😂 

```suggestion
    - docker-py >=3.6.0
```

---

there is some mess with this dependency. 
in pypi there is docker and docker-py
docker-py is old one and apparently, they did not update their conda recipe

---

AFAICS `docker-py` on conda-forge is what `docker` is on PyPI, it is also uptodate with the `docker` pypi package version. So using this should be the right thing.